apps/mobile/src/screens/Home/utils/portfolio.ts

```diff
@@ -96,7 +96,9 @@ export const batchLoadProjects = async (
       if (isTestnet) {
         return testOpenapi.getProtocol({ addr: user_id, id });
       } else {
-        return openapi.getProtocol({ addr: user_id, id });
+        return openapi.getProtocol({ addr: user_id, id }).catch(err => {
+          console.error('[getProtocol] error: ', err, user_id, id);
+        });
```

Comment:
![img_v3_02jr_cd6d6830-ebcb-4921-b7d6-2040de99eafh](https://github.com/user-attachments/assets/fd582271-fd99-4910-900c-ccaa4584d20b)
this will make promise always be `fufilled`